### DISCUSSION TITLE
Argocd UI Is not giving option to sync. Application status is unknown.

### DISCUSSION THREAD

**Discussion by aman35** [argoproj/argo-cd#8708] [Q&A]
Hi, 
We are facing strange issue with argocd.  Our Application Sync is failing with one error and I tried to sync application manually. I realised it is not listing any object (UI) under SYNCHRONIZE RESOURCES:. This section is blank. 
We are using apps of app approach. Now, My argocd is broken. Error which I see is.

ComparisonError: failed to sync cluster https://cluster_URL : failed to load initial state of resource kserve.io: Internal error occurred: error resolving

I am totally clueless.. I have tried restart of argocd-application-controller multiple times.

**Comment by todaywasawesome** [comment]
How are resources? CPU, memory on the cluster? How many applications are there? I would start there.

**Reply by aman35** [reply]
Applications were running fine.

**Comment by MedMokhtarAmmar** [comment]
@aman35  i'm facing the same problem after upgrading to v2.1.2  (previous version 1.7.8) . did you find any solution ?

**Reply by aman35** [reply]
It was due to i compatibility of argocd version 1.1 with kubernetes 1.22. K8s 1.22 has deprecated some apis which are used in argocd 1.1. So we have to upgarde argocd image to version 1.2 which fixed the issue
